src/plugins/intel_cpu/tests/functional/utils/filter_cpu_info.hpp

```diff
@@ -11,4 +11,8 @@ namespace CPUTestUtils {
 std::vector<CPUSpecificParams> filterCPUInfo(const std::vector<CPUSpecificParams>& CPUParams);
 std::vector<CPUSpecificParams> filterCPUInfoForArch(const std::vector<CPUSpecificParams>& CPUParams);
 std::vector<CPUSpecificParams> filterCPUInfoForDevice(const std::vector<CPUSpecificParams>& CPUParams);
+// TODO Split test case of X86_64 and RISCV64
+#if defined(OPENVINO_ARCH_X86_64) || defined(OPENVINO_ARCH_RISCV64)
+std::vector<CPUSpecificParams> filterCPUInfoForDeviceWithFP16(const std::vector<CPUSpecificParams>& allParams);
+#endif
```

Comment:
What is the reason of defining this function for `RISCV64` build?

Reply:
when I started to do this PR,  all the test case of different platform were using same cpp, haven't split into different directory like common, x64. I will check if can only define with condition OPENVINO_ARCH_X86_64 now

Reply:
updated, only define condition OPENVINO_ARCH_X86_64
about test case on RISCV64, I modified the CMakeList.txt. it make clear that which test case will be test on RISCV64. please help to review, thanks